### ISSUE TITLE
Compile-time warning in Elixir 1.9

### ISSUE THREAD

**Issue by ghost** [ravernkoh/gollum#2]
Compiling Gollum with Elixir 1.9 shows the following warning:

```
warning: function init/1 required by behaviour GenServer is not implemented (in module Gollum.Cache).

We will inject a default implementation for now:

    def init(init_arg) do
      {:ok, init_arg}
    end

You can copy the implementation above or define your own that converts the arguments given to GenServer.start_link/3 to the server state.

  lib/gollum/cache.ex:1: Gollum.Cache (module)

Generated gollum app
```

**Comment by ravern** [comment]
Fixed and published as `v0.3.3`.

**Comment by ghost** [comment]
thank you! and thank you for writing + maintaining this library. i use it in a web crawler and the caching behavior is really great!

— Tao
